### PR TITLE
Add templated email to message API

### LINKED ISSUE #122: Backend - Email templating for sendmessage Endpoint
related to #115

### PR DESCRIPTION
**Pull request by ng95junwei**
Fixes #122

### CODE REVIEW

**Comment by subhransu** [comment]
Screenshot 2020-09-02 at 9 49 07 PM

I am using the same image for the sender and receiver so ignore the image. But the title and subtitle of the sender's profile should be displayed right?

File: internal/message/repository.go [hunk 1]
```diff
@@ -44,10 +62,29 @@ func (repo *messageRepository) SendMessage(ctx context.Context, toPersonID strin
 	from := mail.NewEmail("JobsGoWhere Message", "no-reply@jobsgowhere.com")
 	to := mail.NewEmail(toPersonName, toPerson.Email)
 
-	plainTextContent := senderName + "has sent a message" + body
-	htmlContent := "<strong>" + senderName + "</strong> has sent a message, <br>" + body
+	emailContent := EmailContent{
+		LogoURL:          os.Getenv("LOGO_URL"),
+		ReceiverImgURL:   toPerson.AvatarURL.String,
+		ReceiverName:     toPersonName,
+		ReceiverPosition: toPerson.R.PersonProfiles[0].ProfileType.String,
```
Comment (ng95junwei) [inline internal/message/repository.go:69]: @subhransu Thanks for helping to generate that screenshot! I'm supposed to inject it in using ReceiverPosition, ReceiverCompany and ReceiverHeadline but I'm not sure if this is the correct way to do it. Could you confirm?

Comment (subhransu) [reply internal/message/repository.go:69]: This is correct, I might have the data missing. Let me verify.

**Review by subhransu — APPROVED** [review]
Looks okay except the logo. We are fixing it later right?

**Comment by subhransu** [comment]
Screenshot 2020-09-08 at 10 54 28 PM

**Comment by ng95junwei** [comment]
> Looks okay except the logo. We are fixing it later right?
Yup. No code change will be required. As long as the prod env has `LOGO_URL` it should be alright.
`LogoURL:          os.Getenv("LOGO_URL"),`

**Pull request merged**